index.bs

```diff
@@ -1815,10 +1815,7 @@ Data truncation will occur when the specified value exceeds the range of the spe
     </div>
     1. Let |descriptor| be a new {{MLOperandDescriptor}}.
         1. Set |descriptor|.{{MLOperandDescriptor/dataType}} to |type|.
-        1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to `undefined`.
-            <div class="note">
-                In the case of a scalar constant, |descriptor|.{{MLOperandDescriptor/dimensions}} is ignored.
-            </div>
+        1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to the empty [=/list=] « ».
```

Comment:
Nit: "an" instead of "the" and no need for « », so just:

`Set |descriptor|.{{MLOperandDescriptor/dimensions}} to an empty [=/list=].`

(Precedent: examples in Infra. Lists can mutate, unlike strings.)